stackstorm/handler.py

```diff
@@ -93,7 +104,32 @@ def _load_config_schemas():
 CONFIG_SCHEMAS = _load_config_schemas()
 
 
-def base(event, context, passthrough=False):
+def base(event, context, passthrough=False, debug=False):
+    # Set up logging
+    # TODO: Allow log level to be specified by the user, default to DEBUG when
```

Comment:
Should you remove this TODO?

Reply:
`debug` is hardcoded atm. I wanted to ask @enykeev on the best way to provide this argument to the handler (it would just utilize existing `--verbose` CLI flag).

I looked at the existing code and it looks like we have a different handler for default and throughput scenario, but wasn't sure that's the correct approach for `debug` parameter.

Reply:
It seems to me that env variable is the best way to set that kind of parameter. The passthrough parameter returns a vastly different output and requires a different set of transformers anyway so it has its own handler. Logging on the other hand should allow to change it on an already deployed lambda.

Reply:
Good point, I agree about that.

As far as environment variables go - can I directly read them using `os.environ` or do I need to read it from the event object?

Reply:
You can read them from os.environ. This is how we did it previously: https://github.com/StackStorm/serverless-plugin-stackstorm/blob/master/stackstorm/handler.py#L148